grdwindinversion/utils.py

```diff
@@ -1,7 +1,28 @@
+import os
+import time
 import logging
 import xsarsea
 
 
+logger = logging.getLogger('grdwindinversion')
+# logger.addHandler(logging.NullHandler())
+logger.setLevel(logging.DEBUG)
+
+console_handler = logging.StreamHandler()
+console_handler.setLevel(logging.DEBUG)
+formatter = logging.Formatter(
+    '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+console_handler.setFormatter(formatter)
+logger.addHandler(console_handler)
```

Comment:
What is the rational of having 2 handlers rather than one?

Reply:
changed

Reply:
also changed in xsarsea to 

``` python
logging.basicConfig(level=logging.INFO,
                    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
logger = logging.getLogger('xsarsea')
```